src/Calendar.jsx

```diff
@@ -404,6 +405,7 @@ Calendar.defaultProps = {
   showNavigation: true,
   showNeighboringMonth: true,
   view: 'month',
+  navigationLabel: label => label,
```

Comment:
navigationLabel is not used in Calendar. In fact, I wouldn't like to add needless processing at all, so I'd remove this defaultProp altogether.